infer/function.go

```diff
@@ -54,12 +53,15 @@ type derivedInvokeController[F Fn[I, O], I, O any] struct{}
 func (derivedInvokeController[F, I, O]) isInferredFunction() {}
 
 func (*derivedInvokeController[F, I, O]) GetToken() (tokens.Type, error) {
-	var f F
-	tk, err := introspect.GetToken("pkg", f)
-	if err != nil {
-		return "", err
-	}
-	return fnToken(tk), nil
+	// By default, we get resource style tokens:
+	//
+	//	pkg:index:FizzBuzz
+	//
+	// Functions use a different capitalization convection, so we need to convert:
+	//
+	//	pkg:index:FizzBuzz
```

Comment:
The capitalization here is the same as above?

Reply:
Fixed in https://github.com/pulumi/pulumi-go-provider/pull/129/commits/4d9db7c28a33a15140381ceb7a9363bb3c825040